### ISSUE TITLE
MacOs install instructions for rpk should accommodate m1

### ISSUE THREAD

**Issue by WesWWagner** [redpanda-data/documentation#714]
The 60 second guide for macos does not make a distinction between intel processors and m1 processors. There are a suggestion by Alex to integrate the execution of arch inside the curl so users get to the right place.

```
To download the rpk binary:

Download the [rpk archive](https://github.com/redpanda-data/redpanda/releases/latest/download/rpk-darwin-amd64.zip) with curl -LO https://github.com/redpanda-data/redpanda/releases/latest/download**/rpk-darwin-amd64.zip.**
Unpack the archive to /usr/local/bin or any location in your $PATH with unzip rpk-darwin-amd64.zip.
```

**Comment by Feediver1** [comment]
Thanks for the update @WesWWagner ! We'll get it in.

**Comment by Deflaimun** [comment]
Taggint this issue as blocked because we're mssing a review from @WesWWagner on #910

**Comment by JakeSCahill** [comment]
fixed in #1034
